usaspending_api/broker/management/commands/fix_load.py

```diff
@@ -0,0 +1,217 @@
+            value_map.update({
+                'location_country_code': None,
+                'country_name': None
+            })
+
+        location_instance_data = load_data_into_model(Location(),
+                                                      row,
+                                                      value_map=value_map,
+                                                      field_map=field_map,
+                                                      as_dict=True)
+
+        (loc, created) = Location.objects.get_or_create(**
+                                                        location_instance_data)
+        loc.save()
+        return loc
+
+    def fix_places_of_performance(self, fpds_broker_data):
+
+        start_time = datetime.now()
+        value_map = {"place_of_performance_flag": True}
+        for index, row in enumerate(fpds_broker_data, 1):
+            if not (index % 10000):
+                logger.info('Place of performance: fixing row {} ({})'.format(
+                    str(index), datetime.now() - start_time))
+            loc = self.location_from_fpds_row(row,
+                                              field_map=pop_field_map,
+                                              value_map=dict(value_map))
+            for txn in TransactionFPDS.objects.filter(
+                    detached_award_procurement_id=row[
+                        'detached_award_procurement_id']).all():
+                txn.transaction.place_of_performance = loc
+                txn.transaction.save()
+                if txn.transaction == txn.transaction.award.latest_transaction:
+                    txn.transaction.award.place_of_performance = loc
+                    txn.transaction.award.save()
+
+    def fix_recipient_locations(self, fpds_broker_data):
+
+        value_map = {"recipient_flag": True}
+        for index, row in enumerate(fpds_broker_data, 1):
+            if not (index % 10000):
+                logger.info('Recipient location: fixing row {} ({})'.format(
+                    str(index), datetime.now() - start_time))
+            for txn in TransactionFPDS.objects.filter(
+                    detached_award_procurement_id=row[
+                        'detached_award_procurement_id']).all():
+                recip = txn.transaction.recipient
+                last_txn = recip.transactionnormalized_set.order_by(
+                    '-action_date')[0]
+                if txn.transaction == last_txn:
+                    loc = self.location_from_fpds_row(
+                        row,
+                        field_map=le_field_map,
+                        value_map=dict(value_map))
+                    recip.location = loc
+                    recip.save()
+
+    def add_arguments(self, parser):
+
+        parser.add_argument('--piid',
+                            nargs='+',
+                            help="PIID of award(s) to reload")
+
+    @db_transaction.atomic
+    def handle(self, *args, **options):
```

Comment:
Is this script reloading FPDS data to fix the zipcode data at the Award/TransactionFPDS level?

Reply:
Oops, this was a half-finished part of a different branch that I apparently included in this PR with the popular git command, `git-fumble`.  Now removed.  Thanks!